mllib-tests/v1p5/src/main/scala/mllib/perf/MLAlgorithmTests.scala

```diff
@@ -576,8 +609,7 @@ class DecisionTreeTest(sc: SparkContext) extends DecisionTreeTests(sc) {
     (splits, categoricalFeaturesInfo_, labelType)
   }
 
-  override def runTest(rdd: RDD[LabeledPoint])
-    : Either[RandomForestModel, GradientBoostedTreesModel] = {
+  override def runTest(rdd: RDD[LabeledPoint]): DecisionTreeModel = {
```

Comment:
Well, for MLlib, we tried to make sure we cached and materialized before starting the timer for training, to be more careful about timing.  An extra pass seems OK to me to do the same for spark.ml.